workflows/parameters/MPI-ESM1-2-HR-pr.yaml

```diff
@@ -4,19 +4,7 @@ jobs: |
       "target": "historical",
       "variable_id": "pr",
       "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "pr", "source_id": "MPI-ESM1-2-HR", "institution_id": "MPI-M", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20190710" },
-      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp370", "table_id": "day", "variable_id": "pr", "source_id": "MPI-ESM1-2-HR", "institution_id": "DKRZ", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20190710" }
-    },
-    {
-      "target": "ssp",
-      "variable_id": "pr",
-      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "pr", "source_id": "MPI-ESM1-2-HR", "institution_id": "MPI-M", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20190710" },
-      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp370", "table_id": "day", "variable_id": "pr", "source_id": "MPI-ESM1-2-HR", "institution_id": "DKRZ", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20190710" }
-    },
-    {
-      "target": "ssp",
-      "variable_id": "pr",
-      "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "pr", "source_id": "MPI-ESM1-2-HR", "institution_id": "MPI-M", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20190710" },
-      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp245", "table_id": "day", "variable_id": "pr", "source_id": "MPI-ESM1-2-HR", "institution_id": "DKRZ", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20190710" }
+      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp126", "table_id": "day", "variable_id": "pr", "source_id": "MPI-ESM1-2-HR", "institution_id": "DKRZ", "member_id": "r1i1p1f1", "grid_label": "gn", "version": "20190710" }
```

Comment:
We're going to want to confirm that we can append SSP126 onto the historical record. I don't think we want this but we're going to want to confirm with a project lead.

Reply:
Great, let's wait for their input! Thanks !